### ISSUE TITLE
Extension host crashes on macos

### ISSUE THREAD

**Issue by akzaidi** [swdotcom/swdc-vscode-musictime#51]
Issue Type: <b>Bug</b>

## Platform Details

- vscode-insiders: Version: 1.54.0-insider
- OS: macos Catalina

Extension version: 2.1.4
VS Code version: Code - Insiders 1.54.0-insider (52838cf6799cc448e738677ec37e86cf62a5bd89, 2021-02-05T05:15:00.813Z)
OS version: Darwin x64 19.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz (16 x 2400)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|32.00GB (0.38GB free)|
|Process Argv|--crash-reporter-id 4c9f8016-7a81-470c-abd3-2955d044420b|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspyt653:30253241
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30208930
vstry914:30244316
pythonvsdeb440:30224570
pythonvsded773:30223139
pythonvspyt600cf:30251589
core-portspanelcf:30233468
coreuntitledfile:30249963

```

</details>

**Comment by xavluiz** [comment]
Ok, thank you for the update. We'll look into this today.
Regards.

**Comment by xavluiz** [comment]
Hi @akzaidi, we apologize for the delay. Is this issue still happening for you in v2.1.12? If so, have you seen any errors in the VSCode developer tools that can help us reproduce this?
Thanks.

**Comment by akzaidi** [comment]
hey @xavluiz - looks like it's working now! just renabled both musictime and codetime and no crashes so far ❤️

**Comment by xavluiz** [comment]
Hi @akzaidi , thank you for the update but I was able to reproduce the issue on the latest version. It looks like its due to using the plugin with an empty project or non-git based project. We'll have this fixed within the day, thanks again for your patience.

**Comment by xavluiz** [comment]
We've reproduced the issue when opening vscode with a non-git based project and fixed the underlying problem. The fix is now in v2.1.13. Please let us know if there's any other issues. Thank you.
